register/models.py

```diff
@@ -66,8 +66,14 @@ class SignupLog(models.Model):
 
 	type = models.CharField(choices=types, null=True, blank=True, max_length=30)
 
+	def get_company_contact_company(self):
+		if self.company_contact == None:
```

Comment:
A cool one-liner here is `'-' if self.company_contact == None else self.company_contact.company.name`

Reply:
Awesome 😎